ImageConverter/ImageConverter/Input.cs

```diff
@@ -0,0 +1,105 @@
+﻿using ImageConverter.Bmp;
+using ImageConverter.Ppm;
+
```

Comment:
also gif

Reply:
I don't have gif

Reply:
you merged gif branch into this one, wtf